wrappers/python/zxing.cpp

```diff
@@ -47,6 +51,30 @@ Result decode(const Image& image, BarcodeFormat format, bool fastMode, bool tryR
 	return decode(image, {format}, fastMode, tryRotate, hybridBinarizer);
 }
 
+Image encode(std::string text, int width, int height, BarcodeFormat format, int margin, int eccLevel) {
+	MultiFormatWriter writer(format);
+	if (margin >= 0)
+		writer.setMargin(margin);
+	if (eccLevel >= 0)
+		writer.setEccLevel(eccLevel);
+
+	auto bitmap = writer.encode(TextUtfEncoding::FromUtf8(text), width, height).toByteMatrix();
+
+	auto bitmapWidth = bitmap.width();
+	auto bitmapHeight = bitmap.height();
+
+	py::array_t<uint8_t> result = py::array_t<uint8_t>(bitmapWidth * bitmapHeight * sizeof(uint8_t));
```

Comment:
* Please use the `Image` type alias here to be consistent with the return type and the rest of the code.
 * Also, I believe there is a more suitable constructor that takes the final shape of the array (`{bitmap.width(), bitmap.height()}`)
 * That `* sizeof(uint8_t)` makes no sens to me here. Since it is equal to 1 it is no bug but misleading.